routes/api/v1.js

```diff
@@ -142,13 +142,12 @@ api.declare({
   scopes:     [['auth:create-client', 'auth:credentials']],
   title:      "Create Client",
   description: [
-    "Create client with given `clientId`, `name`, `expires`, `scopes` and",
+    "Create a client with given `clientId`, `name`, `expires`, `scopes` and",
     "`description`. The `accessToken` will always be generated server-side,",
     "and will be returned from this request.",
     "",
-    "**Required scopes**, in addition the scopes listed",
-    "above, the caller must also posses the all the scopes that is given to",
-    "the client that is created."
+    "**Required scopes**: in addition the scopes listed",
+    "above, the `scopes` parameter must satisfy the caller's scopes."
```

Comment:
I think "the `scopes` parameter must satisfy the caller's scopes" got flipped around.

```
  "**Required scopes**: in addition the scopes listed",
  "above, the caller must satisfy all scopes granted to the client."
```
or if you must:
```
  "**Required scopes**: in addition the scopes listed",
  "above, the caller must satisfy all scopes from the `scopes` property of the payload."
```